packages/@o3r/design/src/core/design-token/design-token-specification.interface.ts

```diff
@@ -0,0 +1,252 @@
+/** Metadata information added in the design token extension for Metadata extraction */
+export interface DesignTokenMetadata {
+  tags?: string[];
+  /** Description of the variable */
+  label?: string;
+  /** Name of a group of variables */
+  category?: string;
+}
+
+/** Design Token Group Extension fields supported by the default renderer */
+export interface DesignTokenGroupExtensions {
+  /** Indicate the file where to generate the token */
+  o3rTargetFile?: string;
+  /**
+   * Indicate that the variable does not need to be generated
+   * If not generated, the generator will used it's targeted value when referred
```

Comment:
Not sure this is clear for the user what will be generated

Reply:
If you have an idea to be more explicit, I take it :).
(Also at this stage we do not know the language that this Token will be generated to)